### DISCUSSION TITLE
Exporting Data to Excel

### DISCUSSION THREAD

**Discussion by giovanna8988** [mifi/lossless-cut#1793] [Support]
Hello! I would like to export the times to Excel, however the file is not saving as a CSV file. 

Below is a picture of the file automatically saved. 
![Saved Video Files](https://github.com/mifi/lossless-cut/assets/152316141/3046d3d1-5f77-4a77-b9ff-b40a2a751336)

Here I tried altering the export to a different file type, but I don't see CSV or anything compatible with Excel as an option.
![Trying to Alter Extension Type](https://github.com/mifi/lossless-cut/assets/152316141/dd26f9d1-e21d-479d-b874-7510a4d4b823)

Thank you!

**Comment by mifi** [comment]
Hi. Try the file - export menu
